conf/modules/collective_tracking_control.xml

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="collective_tracking_control" dir="ctc">
+  <doc>
+  <description>Distributed algorithm for tracking a velocity/position reference signal. This module needs of custom messages in messages.xml. Please, check the wiki https://wiki.paparazziuav.org/wiki/Module/collective_tracking_control for more information about the messages and how to tune the gains.
```

Comment:
When the doc is generated, the first line is used as "short description" and the rest as "detailed description". Can you separate the first line from the rest ?

Reply:
Oh, I did not do that in my last push. I will make the change and push it.